### ISSUE TITLE
Questions about the training datasets

### ISSUE THREAD

**Issue by lajipeng** [MIDA-group/CoMIR#50]
Hello! Thanks for your great contributions!
However, it seems that there is only evaluation datasets. E.g. how can we get the trainning datasets of Zurich?

**Comment by lajipeng** [comment]
I noticed that your published datasets were divided into 3 sub-groups, which can enable 3-folded cross-validation, but it was still unclear to me that how to get the training datasets. I would appreciate it if you could help me !

**Comment by npielawski** [comment]
Hello Alan!

Thanks for your interest!

The link is indeed hard to find; it is in the first cell of the notebook: https://github.com/MIDA-group/CoMIR/blob/master/train-zurich.ipynb

The link to the Zurich dataset is: https://drive.google.com/file/d/0B9xP9Y5JKJz0ay1HdG4tRm1mME0/view?usp=sharing
Feel free to use our dataloader in the 5th and 6th cells of the notebook above.

Note: you do not need to use the `ground_truth` data as it contains segmentation masks, unless you want to work with masks in your research. Your call 😉 

Feel free to reopen if you have an issue.
